examples/forall-param-reductions.cpp

```diff
@@ -146,6 +146,62 @@ int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
   // _reductions_raja_seq_end
 
 
+//
+// Define RefLoc Type
+//
+
+  using REF_INT_SUM = RAJA::expt::ValOp<int, RAJA::operators::plus>;
+  using REF_INT_MIN = RAJA::expt::ValOp<int, RAJA::operators::minimum>;
+  using REF_INT_MAX = RAJA::expt::ValOp<int, RAJA::operators::maximum>;
+  using REFLOC_INT_MIN = RAJA::expt::ValLocOp<int, RAJA::operators::minimum>;
```

Comment:
We should have an option to change the loc type.

Reply:
Do you mean you want something like `ValLocOp<double, int, minimum>`, where `int` represents the loc type? I don't think we've had that in the past.

Reply:
yep